nbconvert/templates/html/basic.tpl

```diff
@@ -209,3 +209,31 @@ var element = $('#{{ div_id }}');
 </script>
 </div>
 {%- endblock -%}
+
+{%- block data_widget_state scoped %}
+{% set div_id = uuid4() %}
+{% set content = output.data['application/vnd.jupyter.widget-state+json'] %}
+<div id="{{ div_id }}"></div>
+<div class="output_subarea output_widget_state {{extra_class}}">
+<script type="tapplication/vnd.jupyter.widget-state+json">
+var element = $('#{{ div_id }}');
+{
+"model_id": "{{content.model_id}}"
+}
+</script>
```

Comment:
Oh. I thought you wanted that `model_id` specifically. Well, thank you for making the problem more specific which turned out to be a great way to get me to think about how to solve this problem.